docker/env/Dockerfile

```diff
@@ -1,15 +1,25 @@
-FROM centos:7
+FROM python:3.8
 
-ADD install-prereqs.sh install-prereqs.sh
-RUN ./install-prereqs.sh docker
-RUN yum update -y \
-    && yum install -y https://repo.ius.io/ius-release-el7.rpm \
-    && yum install -y python36u python36u-libs python36u-devel python36u-pip
+RUN DEBIAN_FRONTEND=noninteractive apt update && \
+    apt install -y git openssl iproute2 sudo apt-transport-https \
+    ca-certificates curl gnupg2 software-properties-common rsync
+
+# Install Docker cli only
+RUN curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add - && \
+    DEBIAN_FRONTEND=noninteractive add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debian $(lsb_release -cs) stable" && \
+    DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y docker-ce-cli  && \
+    sudo groupadd docker && \
+    sudo usermod -aG docker `whoami`
 
 ADD requirements-python.txt requirements-python.txt
-RUN pip3 install setuptools==40.8.0 pip==19.1.1
+RUN pip3 install setuptools==46.4.0 pip==20.1.1
 RUN pip3 install -r requirements-python.txt
 
-ENV LANG en_US.UTF-8
-ENV LC_ALL en_US.UTF-8
+# update locales
+RUN apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y locales \
+    && sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen \
+    && dpkg-reconfigure --frontend=noninteractive locales \
+    && update-locale LANG=en_US.UTF-8
```

Comment:
Why we need to update locales? It's set to C.UTF-8 and probably we don't need to change it.

Reply:
we had those before, and it was keep supplying warnings,
I've looked for the proper way of setting it, but you are right that this should be the default anyhow.

I'll remove